gateway/src/apicast/proxy.lua

```diff
@@ -264,15 +293,22 @@ function _M:rewrite(service)
   local var = ngx.var
 
   -- save those tables in context so they can be used in the backend client
-  ctx.usage = usage_params
+  context.usage = context.usage or Usage.new()
```

Comment:
This could be provided by `export`, right?

Reply:
Yes. Unfortunately, there's a bug in export(). It does not work correctly on service-level policies.
Something we'll need to fix soon :)